packages/acot-preset-wcag/docs/rules/link-has-name.md

```diff
@@ -22,10 +22,4 @@ Link MUST has name.
 
 ```html
 <a href="https://www.w3.org/TR/WCAG21/#name-role-value"></a>
-
-<style>
-  a::after {
-    content: 'WCAG 2.1 - 4.1.2 Name, Role, Value';
-  }
-</style>
```

Comment:
How about below test case as a bad example?

```html
<a href="https://github.com/acot-a11y/acot">
  <svg
    width="44"
    height="44"
    viewBox="0 0 44 44"
    fill="none"
    role="img"
  >
    <path
      d="M18.5137 40.0256H24.9376C25.7724 40.0256 26.5557 39.6184 27.0409 38.9324L39.1913 21.7523H32.7673C31.9326 21.7523 31.1492 22.1595 30.664 22.8455L18.5137 40.0256Z"
      fill="#2051CD"
    />
    <path
      fill-rule="evenodd"
      clip-rule="evenodd"
      d="M35.873 19.348H32.1184C31.0119 19.348 29.9735 19.8837 29.3304 20.7864L15.7265 39.8794C6.85643 38.7471 0 31.1706 0 21.9928C0 12.0335 8.07355 3.95997 18.0328 3.95997C27.0936 3.95997 34.5936 10.6426 35.873 19.348Z"
      fill="#2051CD"
    />
    <path
      d="M29.0929 40.0256H33.7241C34.3259 40.0256 34.8906 39.7363 35.2404 39.2488L44 27.042H39.3688C38.767 27.042 38.2022 27.3312 37.8525 27.8187L29.0929 40.0256Z"
      fill="#2051CD"
    />
  </svg>
</a>
```

Reply:
Cool, this test case is close to the actual use-case and will be helpful to our users :ok_hand: